egs/swbd/s5c/local/nnet3/multi_condition/run_ivector_common.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash 
+#set -e
+# This script is based on local/nnet3/run_ivector_common.sh.
+# It reverberates the original data with simulated room impulse responses
+
+. cmd.sh
+
+stage=1
+num_data_reps=1  # number of reverberated copies of data to generate
+clean_data_dir=train_nodup
```

Comment:
I don't like the fact that your script changes this variable by adding _sp to it, because it could mislead a reader into thinking that they understand what the variable is.
Better to use different variable names-- you could call this 'input_data_dir', and have 'clean_data_dir' be either $input_data_dir or ${input_data_dir}_sp [if/else].